Sources/FoundationInternationalization/Locale/Locale_ICU.swift

```diff
@@ -1221,7 +1221,7 @@ internal final class _LocaleICU: _LocaleProtocol, Sendable {
                 }
 
                 // Check prefs
-                if let firstWeekdayPref = prefs?.firstWeekday {
+                if prefs?.firstWeekday != nil {
```

Comment:
I looked at this one a while ago and couldn't decide if the error was not comparing with nil or not using the value from prefs. It might be best to split it out.